src/lib/systemlib/system_params.c

```diff
@@ -44,7 +44,7 @@
  *
  * @reboot_required true
  * @min 0
- * @max 99999
+ * @max 9999999
```

Comment:
Why?

Reply:
I don't see why we should limit this. When I create custom airframes that shouldn't go upstream I would like to be free in choosing higher numbers so that I don't get conflicts with upstream in the future

Reply:
That's fine I just didn't see what it had to do with mavlink. You might as well limit it to INT32 max if we're not going to have well defined categories.